portfolio/src/main/webapp/index.html

```diff
@@ -10,5 +10,59 @@
     <script src="script.js" defer></script>
   </head>
   <body>
+    <!--NAV BAR-->
+    <nav>
+      <ul>
+        <li><a href="#home">Home</a></li>
+        <li><a href="#work">Work</a></li>
+        <li><a href="#about">About</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+
+    <!--HOME-->
+    <section id="home">
+      <h1 id="greeting">Hi there!</h1>
+      <p>I am Lin Wei.</p>
+      <!--TODO: include action verbs and animate it using js-->
+      <p>I <span id="actions"></span></p>
+    </section>
+
+    <!--FEATURED WORKS-->
+    <section id="works">
+      <div class="title">
+        <h1>Featured Works</h1>
+      </div>
+      <!--CONTAINER FOR ALL PROJECTS-->
+      <div id="projects">
+
+      </div>
+    </section>
+
+    <!--ABOUT ME-->
+    <section id="about">
+      <div class="title">
+        <h1>About</h1>
+      </div>
+      <div>
+        <p>Some text about me.</p>
+      </div>
+    </section>
+
+    <!--CONTACT-->
+    <section id="contact">
+      <div class="title">
+        <h1>Contact</h1>
+      </div>
+      <div>
+        <p>Drop a comment here to say hi!</p>
```

Comment:
nit: A TODO here too?

Reply:
It's to accommodate next week's project. 😅